static/docs/command-reference/checkout.md

```diff
@@ -21,34 +21,31 @@ which instance of each data file or directory should be used, with the checksums
 saved in the `outs` field. The `dvc checkout` command updates the workspace data
 to match with the <abbr>cached</abbr> files that correspond to those checksums.
 
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
+When using Git, the DVC-files can be kept under version control. DVC-files in
```

Comment:
can be -> are

Reply:
Hmm, you don't HAVE to version them. But I rephrased this to avoid the can vs. is question.

Reply:
In 99.999% cases they are versioned, so no reason to complicate language here, I believe.

Reply:
That's high confidence! 😋 yes, I've already changed this sentence completely. It now starts:

> When using Git, the different DVC-files versioned in separate branches or tags...

Will continue conversation about this paragraph in https://github.com/iterative/dvc.org/pull/864#pullrequestreview-337893478.